gateway/1.5/modules/ROOT/pages/policies-included-traffic-management.adoc

```diff
@@ -0,0 +1,154 @@
+
+| `rules`
+| Optional
+| Empty
+| Array of rulesets for this route. +
+Each route can support multiple rulesets.
+
+| `rules.path`
+| Optional
+| Empty
+| The request path that the route can service. +
+You can define only one “URI Template Regex" path for each ruleset. Only requests with the defined path are sent to this route.
+
+| `rules.methods`
+| Optional
+| Empty
+| Array that defines the types of request methods that the route can service. +
+You can select multiple methods for each ruleset. Only requests that are one of the defined methods are sent to this route. Supported values are `GET`, `POST`, `PUT`, `PATCH`, `DELETE`, `OPTIONS`, `HEAD`, `TRACE`, and `CONNECT`.
+
+| `rules.headers`
+| Optional
+| Empty
+| Array that defines what headers and regular expression value must be present for this route to service the request. +
+Only requests that meet all of the specified header requirements are sent to this route. Additional headers present in the request that are not specifically defined in the rules are ignored.
+
+| `rules.host`
+| Optional
+| Empty
+| The request host URL that the route can service. +
+You can define only one host URL for each ruleset. Only requests made from the defined host are sent to this route.
+
+|===
+
+== How This Policy Works
+
+The Traffic Management policy directs traffic to different sets of upstream services by using different routes with defined rulesets and route order. 
+
+In the following diagram, different routes manage requests to flight information databases and to a customer service application. Route one has two upstream services defined, which direct 70% of requests to a stable database and 30% of requests to a beta database.
+
+image:multiple-upstreams.png[Flex Gateway manages the traffic to multiple upstream services.]
+
+To achieve this configuration, bind two `route-weighted` resources to the exposed API instance. One `route-weighted` resource directs traffic to the flight information database, and the other `route-weighted` resource directs traffic to the customer service application.
+
+=== Upstream Services
+
+You can use multiple upstream services in a single route to direct requests to similar services. For example, to test the performance of a new beta upstream service without sending all traffic to the new service, you can direct half of the traffic to a stable upstream service and half to the new upstream service. 
+
+What upstream service within the route each request is sent to is random and independent of any previous request. The `routes.weight` value defines the chance of a request being sent to a particular upstream service. The percentage of traffic sent to that upstream service is the upstream's `routes.weight` divided by the total weight of all upstream services. For example, if there are two upstream services with weight `1`, each receives 50% of the traffic to that route. If the weight values are `1` and `3`, one route would receive 25% of traffic and the other 75%.
```

Comment:
Suggestion: 
The upstream service within the route that each request is sent to...